spi/common/json-ld-spi/src/main/java/org/eclipse/edc/jsonld/spi/transformer/AbstractJsonLdTransformer.java

```diff
@@ -106,20 +123,55 @@ protected Object transformGenericProperty(JsonValue value, TransformerContext co
      * @param context the transformer context
      */
     protected void transformString(JsonValue value, Consumer<String> resultFunction, TransformerContext context) {
+        resultFunction.accept(transformString(value, context));
+    }
+
+    /**
+     * Transforms a JsonValue to a string and applies the result function. If the value parameter
```

Comment:
Nit: for most of the new methods the Javadoc says that the result function is applied, which is not applicable anymore.